schema/deploy/mutations/generate_reports@1.0.0-rc.5.sql

```diff
@@ -0,0 +1,68 @@
+-- Deploy cif:mutations/generate_reports to pg
+
+begin;
+
+create or replace function cif.generate_reports(revision_id int, report_type text, start_date timestamptz, end_date timestamptz)
+  returns setof cif.form_change as $generate_reports$
```

Comment:
Just a line break for consistency with how we've written those in the past

```suggestion
returns setof cif.form_change as 
$generate_reports$
```